README.md

```diff
@@ -17,51 +18,67 @@ sudo ./bb8/setup_starport.sh /mnt/data/starport
 That's it. Note that this will create a `bb8` user and a symlink to the actual starport at `/var/lib/bb8/starport`
```

Comment:
Ooh, this is wrong! it should be `sudo ./bb8/setup_starport.sh /mnt/data/starport montagu` and it will not create a `bb8` user, it will create a symlink `~/starport` in the given user's home directory

Reply:
it's also `setup_starport` !